be/test/common/config_test.cpp

```diff
@@ -210,4 +380,17 @@ TEST_F(ConfigTest, test_read_write_mutable_string_concurrently) {
     }
 }
 
+TEST_F(ConfigTest, test_alias) {
+    CONF_mInt16(cfg_int32, "8000");
+    CONF_Alias(cfg_int32, cfg_int32_alias);
```

Comment:
what if the alias is defined before the real config item?

Reply:
There be a compile error

Reply:
can it be supported? e.g., add the relative forward declaration for the alias.

do we wish to support so?

Reply:
IMHO, this may not be a common usage((defining alias config before true config)), perhaps support could be added when there is such a need?

Reply:
we can add comment to CONF_Alias, say that it must be defined after the target confvar.

Reply:
> we can add comment to CONF_Alias, say that it must be defined after the target confvar.

OK

Reply:
Comment added, PTAL